### ISSUE TITLE
gRPC stream example?

### ISSUE THREAD

**Issue by belsakn** [hashicorp/go-plugin#158]
Hey, is it possible to open a stream to a plugin? 
I didn't find any example for stream in https://github.com/hashicorp/go-plugin/tree/master/examples

**Comment by belsakn** [comment]
Is this repo still being maintained?

**Comment by mitchellh** [comment]
Yes, we use this project actively in all of our tools that are downloaded tens of millions of times per month. If there are significant bugs we will address them quickly, but unfortunately our examples/docs are lacking and we don't really prioritize those at the moment with other things going on. Sorry!

**Comment by jromero** [comment]
I'm looking for something similar. I'm looking to determine whether this library supports the use of `io.Reader` return types in a protobuf service and if so how. AFAIK, protobuffers don't have any support for streaming so it would be good to see how it's achieved via this library.

**Comment by hessonsu** [comment]
https://github.com/hashicorp/go-plugin/blob/master/internal/plugin/grpc_broker.proto
